core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -242,7 +242,7 @@ class HttpGoogleServicesDAO(
                |""".stripMargin.getBytes))
           // use an object name that will always be superseded by a real storage log
           val storageObject = new StorageObject().setName(s"${bucketName}_storage_00_initial_log")
-          val objectInserter = getStorage(getBucketServiceAccountCredential).objects().insert(getStorageLogsBucketName(googleProject), storageObject, stream)
+          val objectInserter = getStorage(getBucketServiceAccountCredential).objects().insert(GoogleServicesDAO.getStorageLogsBucketName(googleProject), storageObject, stream)
```

Comment:
why did we need to add `GoogleServicesDAO` here?

Reply:
Because we moved this function into a companion object so it could be used without requiring an instance of the `HttpGoogleServicesDAO`